src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.skia.cs

```diff
@@ -150,7 +150,14 @@ internal float GetComputedLineHeight()
 			}
 			else
 			{
-				var font = FontDetailsCache.GetFont(FontFamily?.Source, (float)FontSize, FontWeight, FontStyle);
+				var font = FontDetailsCache.GetFont(FontFamily?.Source, (float)FontSize, FontWeight, FontStretch, FontStyle);
+				if (font.CanChange)
+				{
+					// While font family itself didn't change, OnFontFamilyChanged will invalidate whatever
+					// needed for the rendering to happen correct on the next frame.
+					font.FontUpdated += OnFontFamilyChanged;
```

Comment:
I don't think we should do it this way, this will make for a lot of FontUpdated regitrations. On Wasm, the lookup is made in reverse, so that we don't pay a runtime price for an unfrequent occurence.